### PR TITLE
Fixes #22207 :  Prepare all help files/text in admingui for release

### LINKED ISSUE #22207: Prepare all help files/text in admingui for release
We need to make the following changes to the help text in the console :

    Fix copyright year.
    Wherever applicable : Change references : "Java EE 7" -> "Java EE 8"
                                        Change references : "glassfish3" -> "glassfish5"
                                        Change references : "Oracle GlassFish Server 3.1" -> "GlassFish Server Open
                                        Source Edition  5.0"

### PR DESCRIPTION
**Pull request by pranjal10**
Fixes #22207.

### CODE REVIEW

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

File: appserver/admingui/jdbc/src/main/help/en/help/docinfo.html [hunk 1]
```diff
@@ -40,18 +40,18 @@
...
-<p>April 2013</p>
+<p>August 2017</p>
 <p>This document provides the reference and task online help topics for the jdbc module of the GlassFish Server Administration Console.</p>
 <p><a id="sthref1" name="sthref1"></a></p>
 <hr />
-<p>GlassFish Server 4.0 JDBC Online Help</p>
+<p>GlassFish Server Open Source Edition 5.0 JDBC Online Help</p>
 
-<p>Copyright &copy; 2013,&nbsp;Oracle and/or its affiliates. All rights reserved.</p>
+<p>Copyright &copy; 2017,&nbsp;Oracle and/or its affiliates. All rights reserved.</p>
```
Comment (yaminikb) [inline appserver/admingui/jdbc/src/main/help/en/help/docinfo.html]: Fix year

File: appserver/admingui/jms-plugin/src/main/help/en/help/docinfo.html [hunk 2]
```diff
@@ -40,18 +40,18 @@
...
-<p>April 2013</p>
+<p>August 2017</p>
 <p>This document provides the reference and task online help topics for the jms module of the GlassFish Server Administration Console.</p>
 <p><a id="sthref1" name="sthref1"></a></p>
 <hr />
-<p>GlassFish Server 4.0 JMS Online Help</p>
+<p>GlassFish Server Open Source Edition 5.0 JMS Online Help</p>
 
-<p>Copyright &copy; 2013,&nbsp;Oracle and/or its affiliates. All rights reserved.</p>
+<p>Copyright &copy; 2017,&nbsp;Oracle and/or its affiliates. All rights reserved.</p>
```
Comment (yaminikb) [inline appserver/admingui/jms-plugin/src/main/help/en/help/docinfo.html]: fix year

File: appserver/admingui/jts/src/main/help/en/help/docinfo.html [hunk 3]
```diff
@@ -40,18 +40,18 @@
...
-<p>April 2013</p>
+<p>August 2017</p>
 <p>This document provides the reference and task online help topics for the jts module of the GlassFish Server Administration Console.</p>
 <p><a id="sthref1" name="sthref1"></a></p>
 <hr />
-<p>GlassFish Server 4.0 JTS Online Help</p>
+<p>GlassFish Server Open Source Edition 5.0 JTS Online Help</p>
 
-<p>Copyright &copy; 2013,&nbsp;Oracle and/or its affiliates. All rights reserved.</p>
+<p>Copyright &copy; 2017,&nbsp;Oracle and/or its affiliates. All rights reserved.</p>
```
Comment (yaminikb) [inline appserver/admingui/jts/src/main/help/en/help/docinfo.html]: fix year

File: appserver/admingui/web/src/main/help/en/help/docinfo.html [hunk 4]
```diff
@@ -40,18 +40,18 @@
...
-<p>April 2013</p>
+<p>August 2017</p>
 <p>This document provides the reference and task online help topics for the web module of the GlassFish Server Administration Console.</p>
 <p><a id="sthref1" name="sthref1"></a></p>
 <hr />
-<p>GlassFish Server 4.0 Web Online Help</p>
+<p>GlassFish Server Open Source Edition 5.0 Web Online Help</p>
 
-<p>Copyright &copy; 2013,&nbsp;Oracle and/or its affiliates. All rights reserved.</p>
+<p>Copyright &copy; 2017,&nbsp;Oracle and/or its affiliates. All rights reserved.</p>
```
Comment (yaminikb) [inline appserver/admingui/web/src/main/help/en/help/docinfo.html]: fix year

File: nucleus/pom.xml [hunk 5]
```diff
@@ -1230,6 +1238,40 @@
...
+                            </replacements>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
     </profiles>
 
     <dependencies>
```
Comment (yaminikb) [inline nucleus/pom.xml]: Unwanted nucleus pom changes are showing up

**Review by yaminikb — CHANGES_REQUESTED** [review]

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

**Review by yaminikb — APPROVED** [review]
Looks fine

**Pull request merged**